dashboard/lib/widgets/task_overlay.dart

```diff
@@ -316,8 +317,12 @@ class TaskOverlayContents extends StatelessWidget {
   }
 
   Future<void> _rerunTask() async {
-    final bool success = await buildState.rerunTask(task);
-    final Text snackBarText = success ? const Text(rerunSuccessMessage) : const Text(rerunErrorMessage);
+    final CocoonResponse<bool> rerunResponse = await buildState.rerunTask(task);
+    final Text snackBarText = rerunResponse.data == null
+        ? Text(rerunResponse.error)
+        : rerunResponse.data
+            ? const Text(rerunSuccessMessage)
+            : const Text(rerunErrorMessage);
     showSnackBarCallback(
```

Comment:
We'll want to update this so the snackbar is red for errors. This can probably hook into an error brook instead https://github.com/flutter/cocoon/blob/f5d0471044e3d66d2e6e1aabbbeeecff5c0c1f5a/dashboard/lib/widgets/error_brook_watcher.dart

Reply:
for sure. I will get this part incorporated.